Update github_repo.md (#245)
matrix-org/matrix-hookshot: docs/usage/room_configuration/github_repo.md

To set up a connection to a GitHub Repository in a new room:

(NB you must have permission to bridge GitHub repositories before you can use this command, see [auth](../auth.html#github).)

1. The bridge will need to either:

>     - Have a GitHub installation registered with the organisation (can be user)

Reviewer: "user" just feels a bit vague here

```suggestion
    - Have a GitHub installation registered with the organisation (or GitHub user account)
```
